dbt_coding_conventions.md

```diff
@@ -30,9 +60,9 @@
 
 * Schema, table and column names should be in `snake_case`.
 * Use names based on the _business_ terminology, rather than the source terminology.
-* Table names should be plurals, e.g. `accounts`.

```

Comment:
Was there a reason we got rid of this? I thought it was canon!

Reply:
I have this under the "Model Naming" section! Removed it because it's repetitive.

Reply:
oh word